### PR TITLE
Hangups 0.4.5

### PR DESCRIPTION
**Pull request by aswan89**
###### Motivation for this change
Adds Google Hangouts terminal client to NixPkgs

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -2287,7 +2287,7 @@ in {
   });
 
   elasticsearch-curator = callPackage ../development/python-modules/elasticsearch-curator { };
-
+
```
Comment (dotlambda) [inline pkgs/top-level/python-packages.nix]: remove this whitespace change

**Review by dotlambda — CHANGES_REQUESTED** [review]
Please add a prefix of `pythonPackages.` to the appropriate commit messages.
Also, the `meta.maintainers` field is required.

**Comment by dotlambda** [comment]
@GrahamcOfBorg eval
@GrahamcOfBorg build python2.pkgs.readlike hangups

**Comment by dotlambda** [comment]
> Ran 0 tests in 0.000s

Please provide an appropriate `checkPhase`s or set `doCheck = false` with a comment saying why you're doing that.
Btw, it's very annoying to have to request the same changes again. Have a look at your previous PR. Also, please just keep this PR open and force push to your branch instead of opening a new one.

**Comment by Mic92** [comment]
@GrahamcOfBorg eval

**Comment by LnL7** [comment]
@GrahamcOfBorg eval

**Comment by Mic92** [comment]
You might want to checkout `git-rebase`, so you don't need to close and re-open pull requests for the same package.

**Comment by aswan89** [comment]
Requested changes have been pushed.

**Comment by dotlambda** [comment]
@GrahamcOfBorg build python2.pkgs.readlike hangups

**Comment by aswan89** [comment]
The added dependencies have had their check phase disabled since the PyPi packages have no tests included.

File: pkgs/applications/networking/instant-messengers/hangups/default.nix [hunk 2]
```diff
@@ -0,0 +1,53 @@
+{stdenv, python3, lib}:
```
Comment (dotlambda) [inline pkgs/applications/networking/instant-messengers/hangups/default.nix]: Either `stdenv` or `lib`.

File: pkgs/development/python-modules/ReParser/default.nix [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{ stdenv, buildPythonPackage, fetchPypi, isPy3k, lib}:
```
Comment (dotlambda) [inline pkgs/development/python-modules/ReParser/default.nix]: Either `stdenv` or `lib`.

File: pkgs/development/python-modules/readlike/default.nix [hunk 4]
```diff
@@ -0,0 +1,21 @@
+{ stdenv, buildPythonPackage, fetchPypi, lib }:
```
Comment (dotlambda) [inline pkgs/development/python-modules/readlike/default.nix]: Either `stdenv` or `lib`.

File: pkgs/top-level/python-packages.nix [hunk 5]
```diff
@@ -11845,6 +11845,10 @@ in {
 
   readme_renderer = callPackage ../development/python-modules/readme_renderer { };
 
+  readlike = callPackage ../development/python-modules/readlike { };
+
+  ReParser = callPackage ../development/python-modules/ReParser { };
```
Comment (dotlambda) [inline pkgs/top-level/python-packages.nix]: Please use `reparser`, also for the directory name: https://github.com/NixOS/nixpkgs/pull/45822

**Review by dotlambda — CHANGES_REQUESTED** [review]

**Comment by dotlambda** [comment]
Hangups seems to also be used as a library, e.g. by Home Assistant. So, it should be added to `pythonPackages`. What do you think?

**Comment by aanderse** [comment]
@aswan89 are you able to address the above comments and continue with this PR?

**Comment by jonringer** [comment]
bump

**Comment by jonringer** [comment]
the user's fork was deleted, and no progress in a year.

closing

**Pull request closed without merging**